package.json

```diff
@@ -2,23 +2,24 @@
   "name": "handsontable-multi-select",
   "version": "0.27.9",
   "description": "Handsontable editor and renderer for multi-select cells",
-  "main": "dist/js/handsontable-multi-select.js",
+  "main": "dist/multi-select.js",
   "repository": "https://github.com/orestisrodriguez/handsontable-multi-select",
   "author": "Orestis Rodriguez <orestis@orestisrodriguez.com>",
   "license": "MIT",
   "scripts": {
     "prepublishOnly": "mkdir -p dist/js && mkdir -p dist/css && babel lib/multi-select.js > dist/js/handsontable-multi-select.js && lessc lib/multi-select.less dist/css/handsontable-multi-select.css",
-    "demo": "rollup -c web/rollup.config.js"
+    "demo": "rollup -c web/rollup.config.js",
+    "livereload": "yarn run rollup --config \"./web/rollup.config.js\" --watch",
```

Comment:
you can remove the livereload line here. If you want livereload, then `yarn demo --watch` will do the trick.